pg-testing/src/main/java/org/folio/tlib/postgres/TenantPgPoolContainer.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.tlib.postgres;
```

Comment:
This package is already used in vertx-lib.
Can you add a module-info.java for each maven submodule and fix all issues to achieve [Java 9+ modularity](https://developer.ibm.com/tutorials/java-modularity-5/)?

Reply:
Two hours and a couple of stackoverflow lookups.. No luck.. WIth a module-info.java with a few requires doesn't cut it as everything this depend on , vertx, okapi, cql-java are all unnamed. This library would be the first in folio to adopt module-info.java and one of the very few on GitHub to do so.

Reply:
OK. I have at least some `requires` that works now, so I this should be done soon.

Reply:
The first step is automatic modules: https://github.com/folio-org/folio-vertx-lib/pull/18

That pull request can be merged into this pull request.